src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/VisualizationCard/RelativeDelegateBar.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface RelativeBarProps {
+  otherExpenses: number;
+  recognizedDelegates: number;
+}
+
+export const RelativeDelegateBar: React.FC<RelativeBarProps> = ({ otherExpenses, recognizedDelegates }) => {
+  const { isLight } = useThemeContext();
+
+  return (
+    <DelegateBar isLight={isLight}>
+      {otherExpenses > 0 && <OtherExpenses isLight={isLight} width={otherExpenses} />}
+      {recognizedDelegates > 0 && <RecognizedDelegates isLight={isLight} width={recognizedDelegates} />}
```

Comment:
the width here should be calculated dynamically and normalized in order to avoid UI issues, both parameters should be the expenses amount of each element respectively, so `OtherExpenses` width should be something like `percentageRespectTo(otherExpenses, total)` where `total` is `otherExpenses + recognizedDelegates`. Similar with the `RecognizedDelegates`